app/feature/feature-payments/src/main/kotlin/com/hedvig/android/feature/payments2/details/PaymentDetailsDestination.kt

```diff
@@ -0,0 +1,260 @@
+            contentColor = MaterialTheme.colorScheme.onTypeContainer,
+          ),
+          underTextContent = null,
+        )
+
+        MemberCharge.MemberChargeStatus.PENDING -> PaymentStatusCard(
+          text = stringResource(id = R.string.PAYMENTS_IN_PROGRESS),
+          icon = Icons.Hedvig.InfoFilled,
+          iconColor = MaterialTheme.colorScheme.infoElement,
+          colors = CardDefaults.outlinedCardColors(
+            containerColor = MaterialTheme.colorScheme.infoContainer,
+            contentColor = MaterialTheme.colorScheme.onInfoContainer,
+          ),
+          underTextContent = null,
+        )
+
+        MemberCharge.MemberChargeStatus.FAILED -> PaymentStatusCard(
+          text = stringResource(
+            id = R.string.PAYMENTS_PAYMENT_FAILED, "-",
+          ),
+          icon = Icons.Hedvig.WarningFilled,
+          iconColor = MaterialTheme.colorScheme.error,
+          colors = CardDefaults.outlinedCardColors(
+            containerColor = MaterialTheme.colorScheme.errorContainer,
+            contentColor = MaterialTheme.colorScheme.onErrorContainer,
+          ),
+          underTextContent = null,
+        )
+
+        MemberCharge.MemberChargeStatus.UNKNOWN -> {}
+      }
+
+      Spacer(Modifier.height(32.dp))
+
+      paymentConnection?.connectionInfo?.let {
+        HorizontalItemsWithMaximumSpaceTaken(
+          startSlot = {
+            Text(stringResource(id = R.string.PAYMENTS_PAYMENT_METHOD))
+          },
+          endSlot = {
+            Text(
+              text = stringResource(id = R.string.PAYMENTS_AUTOGIRO_LABEL),
+              textAlign = TextAlign.End,
+              modifier = Modifier.fillMaxWidth(),
+              color = MaterialTheme.colorScheme.onSurfaceVariant,
+            )
+          },
+          modifier = Modifier.padding(vertical = 16.dp),
+        )
+        Divider()
+
+        HorizontalItemsWithMaximumSpaceTaken(
+          startSlot = {
+            Text(stringResource(id = R.string.PAYMENTS_ACCOUNT))
+          },
+          endSlot = {
+            Text(
+              text = it.displayValue,
+              textAlign = TextAlign.End,
+              modifier = Modifier.fillMaxWidth(),
+              color = MaterialTheme.colorScheme.onSurfaceVariant,
+            )
+          },
+          modifier = Modifier.padding(vertical = 16.dp),
+        )
+        Divider()
+
+        HorizontalItemsWithMaximumSpaceTaken(
+          startSlot = { Text(stringResource(id = R.string.PAYMENTS_DUE_LABEL)) },
+          endSlot = {
+            Text(
+              text = it.displayName,
+              textAlign = TextAlign.End,
+              modifier = Modifier.fillMaxWidth(),
+              color = MaterialTheme.colorScheme.onSurfaceVariant,
+            )
+          },
+          modifier = Modifier.padding(vertical = 16.dp),
+        )
+        Divider()
+      }
+    }
+  }
+}
+
+@Composable
+@HedvigPreview
+internal fun PaymentDetailsScreenPreview() {
```

Comment:
```suggestion
private fun PaymentDetailsScreenPreview() {
```
If you did end up adding the live template for this, try changing it to `private` since all previews don't need to be referenced by anything else anywhere.